Cargo.toml

```diff
@@ -14,6 +14,7 @@ members = [
     'examples/gallery',
     'examples/imagefilter',
     'examples/memory',
+    'examples/opengl_underlay',
```

Comment:
what do you feel about dashes?

Reply:
Oh I broke convention - indeed. I'll fix that :)

Reply:
On second thought, I think we have a lot more `_` than `-` (see under `tools` and `examples`, like `fancy_demo`, `slide_puzzle`, etc. - `iot-dashboard` is the odd one out).

Reply:
... `printerdemo_old`

but I think that the underscores are legacy...

Anyway, up to you.